src/test/java/uk/gov/pay/adminusers/client/ledger/service/LedgerServiceTest.java

```diff
@@ -76,4 +78,30 @@ public void getTransaction_shouldDeserialiseLedgerPaymentTransactionCorrectly()
         assertThat(transaction.getTransactionId(), is(externalId));
         assertThat(transaction.getReference(), is("test event ref"));
     }
+
+    @Test
+    public void searchTransactions_shouldDeserialiseSearchTransactionsResponseCorrectly() throws JsonProcessingException {
+        String externalId = "e8eq11mi2ndmauvb51qsg8hccn";
+        ImmutableMap<String, Object> transactionData = ImmutableMap.of(
+                "total", 1,
+                "count", 1,
+                "results", List.of(
+                        Map.of("reference", "ref-1", "created_date", "2023-10-09T16:31:13.511Z",
+                                "transaction_id", externalId, "gateway_account_id", "1")
+                ));
+        String ledgerPayload = new Gson().toJson(transactionData);
```

Comment:
We can use the objectMapper class field instead of instantiating a new Gson object here.

Reply:
Updated!